uwuno/src/main/java/com/learning/uwuno/player.java

```diff
@@ -14,6 +15,8 @@
     final private String pid;
     private String name;
     private ArrayList<card> cardList = new ArrayList<card>(); // TODO: Start game must check to make sure this value is not empty
+    @JsonIgnore
```

Comment:
We can use @JsonIgnoreProperties(value = { "pid", "curDeck"}) above the class declaration to ignore more than 1 field. Test to make sure. https://www.baeldung.com/jackson-ignore-properties-on-serialization